geneselection/datasets/dataset.py

```diff
@@ -2,55 +2,75 @@
 import pandas as pd
 import torch
 from torch.utils.data.dataset import Dataset
+from torch import Tensor
 from collections import Iterable
 from ..utils.dataloader import default_collate
+from typing import Dict, List, Union
 
 
-def gsdataset_from_anndata(adata: anndata.AnnData):
-    return GSDataset(
-        X=torch.from_numpy(adata.X), obs=adata.obs, var=adata.var, uns=adata.uns
-    )
+class GSDatasetVarMismatchError(Exception):
+    """Exception for when asked to merge 2 GSDatasets with different var classes (columns)"""
+    pass
 
 
 class GSDataset(Dataset):
     def __init__(
-        self, X=torch.zeros(1, 1), obs=pd.DataFrame([0]), var=pd.DataFrame([0]), uns={}
+        self,
+        x: Tensor=torch.zeros(1, 1),
```

Comment:
why did  X -> x here?

Reply:
I know it's stupid but the Pep standard complains about uppercase variables. To me, it makes sense Uppercase for matrix but having the IDE highlight it was annoying.